src/data/data.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "src": "https://global-uploads.webflow.com/63994dae1033718bee6949ce/649f3f286772ee18775d26f1_mby979eTPSDlRpr7H2MNSly1Sk-p-500.png",
+    "alt": "Drippi.ai",
```

Comment:
`alt` can be hardcoded with some generic picture, perhaps the main HOW logo, no need to get it from the data source, doesnt make sense to define it there